python/servo/build_commands.py

```diff
@@ -425,6 +426,15 @@ def build_geckolib(self, with_gecko=None, jobs=None, verbose=False, release=Fals
 
         print("GeckoLib build completed in %s" % format_duration(elapsed))
 
+        if with_gecko is not None and ret == 0:
+            print("Copying binding files to style/gecko_bindings...")
+            build_path = path.join(geckolib_build_path, "release" if release else "debug", "build")
+            target_style_path = find_dep_path_newest("style", build_path)
+            out_gecko_path = path.join(target_style_path, "out", "gecko")
```

Comment:
That is not supposed to happen, because we checked that the build has ended successfully, the corresponding directory is supposed to exist. If it doesn't exist, it is indeed a bug.